src/test/java/PageObjectStatic/PageStatic/HomePageStatic.java

```diff
@@ -0,0 +1,54 @@
+package PageObjectStatic.PageStatic;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import java.util.ArrayList;
+
+public class HomePageStatic {
+    WebDriver driver;
+
+    private static By buttonRubberDarkLocator = By.xpath("//nav[@id='site-menu']/ul/li[@class='category-1']");
+    private static By buttonDeliveryInformationLocator = By.cssSelector("#site-menu > ul > li");
+    private static By footerModeLocator = By.cssSelector("#breadcrumbs > ul > li");
+    private static By buttonTermConditionLocator = By.xpath("//li[@class='page-4']");
+    private static By titlePageRubberDarkLocator = By.xpath("//div[@id='box-category']/h1[@class='title']");
+
+
+    public static String duckClickRubberDark(WebDriver driver){
+
+        driver.findElement(buttonRubberDarkLocator).click();
+        return driver.findElement(titlePageRubberDarkLocator).getText();
```

Comment:
Why are we returning element texts in the action-related methods? This may seem clear now, but over the growth of the project it will become not obvious.
Please use separate methods for actions and getting headers/footers/titles etc.
Also it will reduce code repeatativity - currently you return a footer in several methods which makes you use the same line in multiple places.